<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_scene</class>
 <widget class="QWidget" name="game_scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="game">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/new/prefix1/game.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="score">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="score_title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ff0000;&quot;&gt;Score&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="time_title">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>45</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ff0000;&quot;&gt;Time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="covered">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>90</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/new/prefix1/chrome_icon.png</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="count">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ball">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/new/prefix1/ball.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="king_ball">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/new/prefix1/king_ball.png</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-16</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLCDNumber" name="fr_score">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="fr_score_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Score</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fr_restart">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fr_exit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="fr_lael">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/new/prefix1/fire_dragon.png</pixmap>
    </property>
   </widget>
   <zorder>fr_lael</zorder>
   <zorder>fr_score_title</zorder>
   <zorder>fr_score</zorder>
   <zorder>fr_exit</zorder>
   <zorder>fr_restart</zorder>
   <zorder>king_ball</zorder>
   <zorder>king_ball</zorder>
  </widget>
  <widget class="QPushButton" name="test_ball_hit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>hit ball</string>
   </property>
  </widget>
  <widget class="QLabel" name="for_ball_left">
   <property name="geometry">
    <rect>
     <x>-270</x>
     <y>10</y>
     <width>47</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="for_ball_right">
   <property name="geometry">
    <rect>
     <x>-230</x>
     <y>10</y>
     <width>47</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_king_ball_hit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>hit king_ball</string>
   </property>
  </widget>
  <widget class="QLabel" name="for_king_left">
   <property name="geometry">
    <rect>
     <x>-210</x>
     <y>80</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="for_king_right">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>80</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>game</zorder>
  <zorder>score</zorder>
  <zorder>score_title</zorder>
  <zorder>time_title</zorder>
  <zorder>covered</zorder>
  <zorder>count</zorder>
  <zorder>ball</zorder>
  <zorder>king_ball</zorder>
  <zorder>frame</zorder>
  <zorder>fr_label</zorder>
  <zorder>test_ball_hit</zorder>
  <zorder>for_ball_left</zorder>
  <zorder>for_ball_right</zorder>
  <zorder>test_king_ball_hit</zorder>
  <zorder>for_king_left</zorder>
  <zorder>for_king_right</zorder>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
